transport/tchannel/channel_transport.go

```diff
@@ -95,12 +96,13 @@ func (options transportOptions) newChannelTransport() *ChannelTransport {
 // If you have a YARPC peer.Chooser, use the unqualified tchannel.Transport
 // instead.
 type ChannelTransport struct {
-	ch     Channel
-	name   string
-	addr   string
-	tracer opentracing.Tracer
-	logger *zap.Logger
-	router transport.Router
+	ch               Channel
+	name             string
+	addr             string
+	tracer           opentracing.Tracer
+	logger           *zap.Logger
+	router           transport.Router
+	forwardingHeader bool
```

Comment:
So perhaps this should be an enum of "writeExactHeaderCase" and "writeCanonicalizedHeaderCase" of type name "writeHeaderCase", or still a bool but called "writeExactHeaderCase" since canonicalized is default.